diff -urNp meson_python-0.15.0.old/pyproject.toml meson_python-0.15.0/pyproject.toml
--- meson_python-0.15.0.old/pyproject.toml	2024-03-28 17:17:48.284880214 -0400
+++ meson_python-0.15.0/pyproject.toml	2024-03-28 17:18:06.768899509 -0400
@@ -7,7 +7,7 @@ build-backend = 'mesonpy'
 backend-path = ['.']
 requires = [
   'meson >= 0.63.3; python_version < "3.12"',
-  'meson >= 1.2.3; python_version >= "3.12"',
+  'meson[ninja] >= 1.2.3; python_version >= "3.12"',
   'pyproject-metadata >= 0.7.1',
   'tomli >= 1.0.0; python_version < "3.11"',
 ]
